src/me/components/Resources.tsx

```diff
@@ -59,6 +60,7 @@ const ResourceLists: FC = () => {
             ctaText="Learn more"
             title="New time-series engine for InfluxDB"
           />
+          <BlogFeed />
```

Comment:
If we feature flag this with a temp flag, we can test first to see if we're running into a CORS issue before folks generally see it in prod.

Reply:
+1 about having a temporary feature flag to test in prod

Reply:
done